packages/libs/deer/LICENSE-MIT.md

```diff
@@ -0,0 +1,22 @@
+MIT License
+===========
+
+Copyright © 2022–, HASH
```

Comment:
This would be inconsistent with all of our other markdown licenses, so I'd suggest not.

If there's a reason this needs changing we should do it globally across licenses (and please DM me).